src/containers/layouts/auth/signIn1/signIn1.component.tsx

```diff
@@ -84,6 +79,10 @@ class SignIn1Component extends React.Component<SignIn1Props, State> {
     this.setState({ formValue });
   };
 
+  private keyboardOffset = (height: number): number => {
+    return height - 152;
```

Comment:
Why exactly 152?

Reply:
Total height of bottom components which can be hidden by keyboard. Currently there is no way to handle it, if not only using `onLayout` (and wrapping it into one container in some cases)